pyproject.toml

```
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools<70", "wheel"]
+requires = ["setuptools<72.0.0", "wheel"]
```

We don't want to pin less than 72, we want to use 72

```suggestion
requires = ["setuptools~=72", "wheel"]
```

---

Local compilation has been installed correctly
Collecting setuptools==72
e8/setuptools-72.0.0-py3-none-any.whl (2.3 MB)

---

Doesn't matter. Please update pyproject.toml to match setup.py.
They should refer to the same setuptools version